recipe/meta.yaml

```diff
@@ -27,6 +36,7 @@ requirements:
     - gsl
     - fftw
     - glpk
+    - suitesparse 4.4.*
```

Comment:
Currently `4.5.*` is not supported so we are pinned to `4.4.*`. As `4.4.*` provides static libraries only and these are what `cvxopt` uses, we don't need `suitesparse` as a run time dependency only as a build time dependency. See this [comment]( https://github.com/cvxopt/cvxopt/issues/58#issuecomment-220971823 ) for details.